File: src/main/java/commands/listeners/OnStaticReactionAddListener.java
package commands.listeners;

import net.dv8tion.jda.api.entities.Message;
import net.dv8tion.jda.api.events.message.guild.react.GuildMessageReactionAddEvent;

public interface OnStaticReactionAddListener {

    void onStaticReactionAdd(Message message, GuildMessageReactionAddEvent event) throws Throwable;

    String titleStartIndicator();

}

File: src/main/java/commands/listeners/OnStaticReactionAddListener.kt
package commands.listeners

import net.dv8tion.jda.api.entities.Message
import net.dv8tion.jda.api.events.message.react.MessageReactionAddEvent

interface OnStaticReactionAddListener {

    @Throws(Throwable::class)
    fun onStaticReactionAdd(message: Message, event: MessageReactionAddEvent)

}